support.html

```diff
@@ -0,0 +1,54 @@
+<!--#include virtual="includes/_header.htm" -->
+<body class="page-contact ">
+<!--#include virtual="includes/_top.htm" -->
+<div class="content">
+	<!--#include virtual="includes/_nav.htm" -->
+	<div class="right">
+		<h1 class="content-title">Support</h1>
+		<h3>Community support</h3>
+
+		<p>
+			This is an open source project so the amount of time community members have available to help resolve your issue
+			is often limited as help is provided on a volunteer basis from
+			<a href="https://www.apache.org/foundation/how-it-works/#hats" target="_blank">individuals</a>.
+			However, it is free, and you are often able to discuss issues with the developers who actually wrote the code
+			you are running.
+		</p>
+		<p>
+			If you want community support then please <a href="https://kafka.apache.org/contact">contact us</a>.
+			If you’re fairly certain you’re hitting a bug please report it via one of our
+			<a href="https://kafka.apache.org/contributing">issue trackers</a>. Be sure to review
```

Comment:
I feel like this should link to JIRA directly, and also mention it by name instead of saying "one of our issue trackers" -- we really only have the one. The "contributing" link would be more appropriate for the following sentence, which does indeed appear to be about contributing rather than reporting issues or "community support". Frankly this section is kind of all over the place but I think it's fine as long as we hit on the important notes -- like where to report issues

Reply:
Ok, I will update to be more specific about the issue tracker.